kanban-all/kanban-data/pom.xml

```diff
@@ -15,6 +15,11 @@
             <artifactId>spring-data-jpa</artifactId>
             <version>1.11.0.RELEASE</version>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>5.2.7.Final</version>
```

Comment:
it's a common technique, you can find BOM model used in Spring project itself, etc.